### DISCUSSION TITLE
Option to disable "[zustand devtools middleware] Please install/enable Redux devtools extension" warning

### DISCUSSION THREAD

**Discussion by FKSI** [pmndrs/zustand#1946] [Ideas]
Hi, 

Although it's now currently possible to disable the warning message with the `enabled` props and/or `process.env.NODE_ENV` values it would be great to consider other cases where you want to have not displayed.

**Comment by dai-shi** [comment]
Would removing devtools entirely help in such cases?

**Reply by FKSI** [reply]
No not really. Let's maybe consider some environment variable so one can disable the warning while not forcing the others to disable the warning.

**Reply by dai-shi** [reply]
I think you should disable the middleware with an environment variable of your choice. Or, if it's just to hide the console warning, you could override console.warn if you can't touch your code.

I don't think there's a reasonable way to do it on the library end.
